### DISCUSSION TITLE
FYI, RetroArch/Libretro devs are working on similar game adapter as open source hardware

### DISCUSSION THREAD

**Discussion by Hedda** [sanni/cartreader#101] [General Chat]
RetroArch/Libretro developers have announced that they are working on a similar adapter which will become open-source hardware:

https://www.libretro.com/index.php/introducing-the-retroarch-open-hardware-project/

https://www.reddit.com/r/RetroArch/comments/lpp528/introducing_the_retroarch_open_hardware_project/

"ProjectFuture" goals sound to be high-speed so could potentially be used for playing games directly from a game cartridge(?).

From the picture posted, it looks to be based on an STMicroelectronics "STM32 F7" which is a 32-bit ARM Cortex-M7 microcontroller(?)

https://www.libretro.com/wp-content/uploads/2021/02/cartridge-adapter-openhardware2.png

https://www.st.com/en/microcontrollers-microprocessors/stm32f7-series.html

If so then I am guessing it is the STM32F767ZI (STM32F7 MCU) and developers prototype using NUCLEO-F767ZI dev board

https://www.st.com/content/st_com/en/products/evaluation-tools/product-evaluation-tools/mcu-mpu-eval-tools/stm32-mcu-mpu-eval-tools/stm32-nucleo-boards/nucleo-f767zi.html

https://www.st.com/en/microcontrollers-microprocessors/stm32f767zi.html

STMicro STM32 Nucleo Boards with STM32 MCUs can ex. be used with ST STM32 ( STSTM32) development platform for PlatformIO:

https://www.st.com/en/evaluation-tools/stm32-nucleo-boards.html

https://github.com/platformio/platform-ststm32

https://docs.platformio.org/en/latest/boards/index.html#st-stm32

https://docs.platformio.org/en/latest/boards/ststm32/disco_f750n8.html

https://www.st.com/content/st_com/en/products/evaluation-tools/product-evaluation-tools/mcu-mpu-eval-tools/stm32-mcu-mpu-eval-tools/stm32-nucleo-boards/nucleo-f767zi.html

https://www.st.com/en/evaluation-tools/stm32f7508-dk.html

https://www.st.com/en/embedded-software/stm32cubef7.html

PS: Cost per STM32F767ZI as SMC looks to cost between $12 to $17 when bought in small quantities to only build your own DYI board.

**Comment by sanni** [comment]
This looks very promising, m4xw is a great developer. Thanks for posting.
